gradle.properties.dist

```diff
@@ -0,0 +1 @@
+game.full_path = C:\\Program Files (x86)\\Fractal Softworks\\Starsector\\mods\\stelnet
```

Comment:
Could this be made more generic, e.g. full path to the game folder (sans `\\mods\\stelnet`) with `mods` defined somewhere else, and modname taken from the `mod_info.json` via slurper?